src/com/sun/ts/tests/signaturetest/javaee/JavaEESigTest.java

```diff
@@ -85,14 +85,14 @@
 
   public static final String KEYWORD_JAVAEE = "ejb interceptors caj jms wsmd javamail"
       + " cdi di beanval persistence jaxb saaj jaxws connector"
-      + " jacc jaspic jsonp jta el servlet jsf jaxrs websocket batch concurrency jsonb securityapi";
+      + " jacc jaspic jsonp jta el servlet jsf jaxrs websocket concurrency jsonb securityapi";
```

Comment:
We should keep current src/com/sun/ts/tests/signaturetest/javaee/JavaEESigTest.java settings also for signature testing.